content/en/apps/guides/messaging/rapidpro.md

```diff
@@ -0,0 +1,114 @@
+---
+title: RapidPro SMS Gateway
+linkTitle: "RapidPro"
+weight:
+description: >
+    Integration for sending and receiving SMS
+relatedContent: >
+    apps/reference/app-settings/sms
+    apps/guides/forms/app-form-sms
+    apps/guides/messaging/sms-states
+    apps/guides/messaging/message-loops
+    apps/guides/messaging/shortcodes
+---
```

Comment:
Similar to [the other comment](https://github.com/medic/cht-docs/pull/462/files#r608814872), RapidPro can be used for messaging beyond SMS too so the title should reflect that too

```suggestion
---
title: RapidPro Messaging Gateway
linkTitle: "RapidPro"
weight:
description: >
    Integration for sending and receiving messages
relatedContent: >
    apps/reference/app-settings/sms
    apps/guides/forms/app-form-sms
    apps/guides/messaging/sms-states
    apps/guides/messaging/message-loops
    apps/guides/messaging/shortcodes
---
```